libs/vkd3d/resource.c

```diff
@@ -631,6 +631,8 @@ static HRESULT vkd3d_get_image_create_info(struct d3d12_device *device,
 
     image_info->mipLevels = min(desc->MipLevels, max_miplevel_count(desc));
     image_info->samples = vk_samples_from_dxgi_sample_desc(&desc->SampleDesc);
+    image_info->tiling = VK_IMAGE_TILING_OPTIMAL;
```

Comment:
Do we check for depth-stencil usage + host image and report a FIXME and E_NOTIMPL?